client/lobby/SelectionTab.h

```diff
@@ -51,14 +61,16 @@ class SelectionTab : public CIntObject
 	std::shared_ptr<CAnimation> iconsLossCondition;
 
 public:
-	std::vector<std::shared_ptr<CMapInfo>> allItems;
-	std::vector<std::shared_ptr<CMapInfo>> curItems;
+	std::vector<std::shared_ptr<ElementInfo>> allItems;
+	std::vector<std::shared_ptr<ElementInfo>> curItems;
+	std::string curFolder;
 	size_t selectionPos;
-	std::function<void(std::shared_ptr<CMapInfo>)> callOnSelect;
+	std::function<void(std::shared_ptr<ElementInfo>)> callOnSelect;
 
 	ESortBy sortingBy;
 	ESortBy generalSortingBy;
 	bool sortModeAscending;
+	int curFilterSize = 0;
```

Comment:
No need to abbreviate variable names
```suggestion
	int currentMapSizeFilter = 0;
```